added convo dials doc (#452)
vocodedev/vocode-python: docs/conversational-dials.mdx

In summary:
- **Possible values:** low, high
- **Default value:** low
- low = ignore non-interrupting words
- high = stop at any word

## Endpointing Sensitivity

`endpointing_sensitivity` controls how the bot determines the when the human is done speaking. It can be one of **sensitive**, **relaxed**, or **auto** (default and currently equivalent to relaxed). 

A bot with `endpointing_sensitivity == 'sensitive'` treats the end of a sentence or a short pause as the end of the human's turn. This works well for quick conversations in which you expect short, single-sentence responses from the human.
A bot with this sensitivity runs the risk of interrupting the human if they continue speaking after a single sentence.

A bot with `endpointing_sensitivity == 'relaxed'` waits a short time after the end of a human's sentence before considering their turn over. Alternatively, when a sentence is not finished, relaxed sensitivity waits for a longer pause before marking the end of the human's speech.
This sensitivity has less risk of interrupting the human but can potentially have a slower response time. Note that auto sensitivity is equivalent to relaxed.

> ***Note:** the amount of time the agent waits during a pause is adjusted based on the human's utterance speed. If the human speaks fast, the bot will wait less, and if the human speaks slow the bot will wait longer.*

Reviewer: ```suggestion
***Note:** the amount of time the agent waits during a pause adapts to the human's utterance speed. If the human speaks fast, the bot will wait less, and if the human speaks slow the bot will wait longer.*
```

In summary:
- **Possible values:** sensitive, relaxed, auto
- **Default value:** auto
- sensitive = end of first sentence or shorter pause
- relaxed/auto = pause after end of first sentence or longer pause

## Conversation Speed

> `conversation_speed` controls when the bot will begin responding to the human after they finish speaking. This includes normal utterances from the human as well as interruptions.

Reviewer: ```suggestion
`conversation_speed` controls the length of endpointing pauses, i.e. how long the bot will wait before responding to the human. This includes normal utterances from the human as well as interruptions.
```
